### ISSUE TITLE
don't report error/code for failed command

### ISSUE THREAD

**Issue by rabbah** [nimbella/nimbella-cli-plugins#7]
Note the last two lines.

```
> nim auth postman --add xxx
Manage Postman API Keys

USAGE
  $ nim auth:postman

OPTIONS
  -a, --add            Add a Postman API key
  ...

 ›   Error: EEXIT: 0
 ›   Code: EEXIT
```

**Comment by bhageena** [comment]
Thanks, it has now been handled and will make into the next npm release.

**Comment by bhageena** [comment]
This has been fixed and is part of release [v0.1.2](https://www.npmjs.com/package/@nimbella/plugin-postman/v/0.1.2) onwards.
